### PR TITLE
announcement bar created

### PR DESCRIPTION
**Pull request by NikhitaDesai**
### Problem
Created announcement bar and header as per the figma.
Monday Card Link :(announcement bar) https://marmeto-force.monday.com/boards/5104690360/pulses/5296291094
Monday Card Link ( header):https://marmeto-force.monday.com/boards/5104690360/pulses/5296291547
Preview link: https://mivi-website.myshopify.com/?preview_theme_id=161424736533
### Solution
Added the relatable CSS properties to match the given figma
image
### Impacted Areas
All pages
### Done Checklist
- [x] Performed unit testing to ensure functionality and identify potential issues.
- [x] Followed basic code standards and best practices. (indentation, media queries, basic customizer things)
- [x] Added comments to explain complex logic or improve code readability.
- [x] Followed Naming conventions and proper variable declaration.
- [x] Handled edge cases and errors gracefully to prevent crashes.
- [x] Optimized the code for production to ensure performance.
Please review this PR and provide feedback. Thank you!

### CODE REVIEW

**Comment by NikhitaDesai** [comment]
@MadhanSamiappan Updated the changes, review the code

File: assets/component-mega-menu.css [hunk 1]
```diff
@@ -1,87 +1,86 @@
...
+  padding-bottom: 26px;
+  padding-top: 39px;
+  color: #4B4955;
+font-family: Gilroy;
+font-size: 14px;
+font-style: normal;
+line-height: 100%; /* 14px */
+letter-spacing: 2.8px;
+text-transform: uppercase;
+text-decoration: none;
```
Comment (MadhanSamiappan) [inline assets/component-mega-menu.css:83]: for properties like font-family ,size you can use the base classess you created in styleguideline right?

File: sections/header.liquid [hunk 2]
```diff
@@ -690,7 +697,35 @@ sticky-header.header-wrapper.color-background-1.gradient.header-wrapper--border-
...
+          {
+         "type":"text",
+          "id":"megamenu_text",
+          "label":"Megamenu Text"
+        },
+        {
+        "type":"image_picker",
+        "id":"mobile_image",
+        "label":"Mobile Image"
+      }
```
Comment (MadhanSamiappan) [inline sections/header.liquid:727]: indents,
image picker info

File: snippets/header-drawer.liquid [hunk 3]
```diff
@@ -57,16 +63,76 @@
...
+           {% comment %}
+                                          {% render 'image' class = 'image-{{ block.id }}' mobileSrc:block.settings.megamenu_image, desktopSrc: block.settings.megamenu_image %}
+                                          {% endcomment %}
+                                      <p>{{ block.settings.mobile-megamenu_text }}</p>
+                                       </div>
+                                    </div>
+                                    {% endif %}
+                                    {% endif %}
+                                    {% endfor %}
+                                 </div>                                   {% endcomment %}
```
Comment (MadhanSamiappan) [inline snippets/header-drawer.liquid:121]: remove commented out code

File: snippets/header-drawer.liquid [hunk 4]
```diff
@@ -57,16 +63,76 @@
...
                                 {%- else -%}
                                   <details id="Details-menu-drawer-{{ link.handle }}-{{ childlink.handle }}">
                                     <summary
                                       id="HeaderDrawer-{{ link.handle }}-{{ childlink.handle }}"
-                                      class="menu-drawer__menu-item link link--text list-menu__item focus-inset"
+                                      class="menu-drawer__menu-item link link--text list-menu__item focus-inset kdfdk"
```
Comment (MadhanSamiappan) [inline snippets/header-drawer.liquid:126]: remove scrap class

File: snippets/header-drawer.liquid [hunk 5]
```diff
@@ -57,16 +63,76 @@
...
                                     >
                                       {{ childlink.title | escape }}
                                       {% render 'icon-arrow' %}
                                       {% render 'icon-caret' %}
                                     </summary>
+
```
Comment (MadhanSamiappan) [inline snippets/header-drawer.liquid:132]: remove extra lines

File: snippets/header-drawer.liquid [hunk 6]
```diff
@@ -88,7 +154,7 @@
                                             <a
                                               id="HeaderDrawer-{{ link.handle }}-{{ childlink.handle }}-{{ grandchildlink.handle }}"
                                               href="{{ grandchildlink.url }}"
-                                              class="menu-drawer__menu-item link link--text list-menu__item focus-inset{% if grandchildlink.current %} menu-drawer__menu-item--active{% endif %}"
+                                              class="menu-drawer__menu-item link link--text list-menu__item focus-inset dkfldk{% if grandchildlink.current %} menu-drawer__menu-item--active{% endif %}"
```
Comment (MadhanSamiappan) [inline snippets/header-drawer.liquid:157]: remove unwanted texts

File: snippets/header-mega-menu.liquid [hunk 7]
```diff
@@ -39,15 +39,42 @@
...
+                           
+                            {% if child_title == level2_title %}
+                              
+                              {% assign headerId = link.title | downcase | replace: ' ', '' %}
+                              {% assign blockId = block.settings.megamenu_link_text | downcase | replace: ' ', '' %}
+                              {% if headerId == blockId %}
+                                <div class="megamenu_images_wrapper">
+                                  <div class="child_link_images font-bold">
+                                    <img
+                                      src="{{ block.settings.megamenu_image | image_url }}"
```
Comment (MadhanSamiappan) [inline snippets/header-mega-menu.liquid:65]: use image snippet instead of putting img tag directly

File: snippets/icon-arrow.liquid [hunk 8]
```diff
@@ -7,4 +7,8 @@
   xmlns="http://www.w3.org/2000/svg"
 >
   <path fill-rule="evenodd" clip-rule="evenodd" d="M8.537.808a.5.5 0 01.817-.162l4 4a.5.5 0 010 .708l-4 4a.5.5 0 11-.708-.708L11.793 5.5H1a.5.5 0 010-1h10.793L8.646 1.354a.5.5 0 01-.109-.546z" fill="currentColor">
-</svg>
+</svg> {% endcomment %}
```
Comment (MadhanSamiappan) [inline snippets/icon-arrow.liquid:10]: remove commented code

File: assets/component-menu-drawer.css [hunk 9]
```diff
@@ -79,6 +79,7 @@ details[open].menu-opening > .menu-drawer__submenu {
 .menu-drawer__inner-container {
   position: relative;
   height: 100%;
+  background: #F6F6F7;
```
Comment (MadhanSamiappan) [inline assets/component-menu-drawer.css:82]: always create a base class for repeating props like bg color here

**Review by MadhanSamiappan — CHANGES_REQUESTED** [review]
refer the comments on files and fix accordingly

File: sections/header.liquid (hunk 2 above)
Comment (NikhitaDesai) [reply sections/header.liquid:727]: I changed this code

**Review by MadhanSamiappan — APPROVED** [review]
Code review - 4.89
not using base classes

**Pull request merged**